sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -162,6 +162,22 @@ private PollerFlux(Duration pollInterval,
         this.syncActivationOperation = cxt -> activationOperation.apply(cxt).block();
     }
 
+    /**
+     * Creates a poller flux instance that returns an error on subscription.
+     *
+     * @param ex The exception to be returned on subscription of this {@link PollerFlux}.
+     * @param <T> The type of poll response value.
+     * @param <U> The type of the final result of long running operation.
+     * @return A poller flux instance that returns an error without emitting any data.
+     *
+     * @see {@link Mono#error(Throwable)}
+     * @see {@link Flux#error(Throwable)}
+     */
+    public static <T, U> PollerFlux<T, U> error(Exception ex) {
```

Comment:
Should this use `Throwable` instead?

Reply:
Initially, I started with Throwable but it conflicts with Flux's `error` method. 
So, cannot use `Throwable` here:
`'error(Throwable)' in 'com.azure.core.util.polling.PollerFlux' clashes with 'error(Throwable)' in 'reactor.core.publisher.Flux'; both methods have same erasure, yet neither hides the other`